ios/Classes/PusherFlutterPlugin.m

```diff
@@ -74,7 +74,11 @@ - (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result
     if ([call.method isEqualToString:@"create"]) {
         NSString *apiKey = call.arguments[@"api_key"];
         NSString *cluster = call.arguments[@"cluster"];
-        self.pusher = [PTPusher pusherWithKey:apiKey delegate:self encrypted:YES cluster:cluster];
+        if ([cluster isEqual:nil] || [cluster isEqualToString:@""]) {
```

Comment:
You can make this part shorter.

`if ([cluster length] > 0)`

Reply:
Nice, good catch.